ivy/functional/frontends/torch/tensor.py

```diff
@@ -208,7 +208,7 @@ def asin_(self):
     def sum(self, dim=None, keepdim=False, *, dtype=None):
         return torch_frontend.sum(self, dim=dim, keepdim=keepdim, dtype=dtype)
 
-    @with_unsupported_dtypes({"2.0.1 and below": ("float16",)}, "torch")
+    @with_unsupported_dtypes({"2.0.1 and below": ("float16", "bfloat16")}, "torch")
```

Comment:
Adding `bfloat16` to `with_unsupported_dtype`s decorator is not the right way to fix this test because `torch` supports `bfloat16` on both the functional and instance implementations of `sin`.

Please remove `bfloat16` from this list